ChemistryLib/PhreeqcKernelData/PurePhaseAssemblage.cpp

```diff
@@ -0,0 +1,38 @@
+/**
+ * \file
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include <algorithm>
+
+#include "PurePhaseAssemblage.h"
+
+namespace ChemistryLib
+{
+namespace PhreeqcKernelData
+{
+PurePhase::PurePhase(std::string&& name,
+                     double const initial_amount,
+                     double const saturation_index)
+{
+    this->name = name;
+    moles = initial_amount;
+    si = saturation_index;
+    si_org = saturation_index;
```

Comment:
should go into initializer list...

Reply:
These parameters belong to the parent class `Phreeqc::cxxPPassemblageComp` and have been initialized in the parent class constructor. I reluctantly wrote in this way.

Reply:
I see. Had a look on the particular class, it indeed has no usable ctors. :roll_eyes: 
Still, could you rewrite the "name" line as
```c++
    name = name_;
```
?

Reply:
changed.